### DISCUSSION TITLE
Gmail is blocking outgoing emails

### DISCUSSION THREAD

**Discussion by toobie83** [dani-garcia/vaultwarden#2820] [Q&A]
Hi there, I configured my Vaultwarden with the email smtp feature. Worked well before the update from bitwarden_rs to vaultwarden or it stopped somewhere.. dont know exactly.
Emails from Ombi for example are going through. 
But vaultwarden emails are going to be blocked with no real reason.
So they will be sent correctly, I can see them in gmail but they are not leaving my inbox/outbox.

Thank you for your help

Tobi.

![-5353001269708570939_121](https://user-images.githubusercontent.com/59822996/195767874-1739d06d-3d94-4c44-9356-1a5ec308ebd5.jpg)

My configuration is the following:
![image](https://user-images.githubusercontent.com/59822996/195768064-e10a76a1-eafd-4abf-a8eb-77a86b3bd006.png)

**Comment by Karunamon** [comment]
I would recommend using something more developer friendly than Gmail for precisely this reason; Gmail cannot be trusted to reliably send or deliver them to the recipient's inbox based on completely opaque criteria.

For a single instance, I have used [Mailjet](mailjet.com), their free tier provides thousands of daily messages and if all you need is an SMTP endpoint to hook the application up to, they will give you that no problem. All you need is a domain name.
